### PR TITLE
feat(db): store user agent and last-access time in sessionTokens

### PR DESCRIPTION
**Pull request by philbooth**
Partially addresses https://github.com/mozilla/fxa-auth-server/issues/983.

Depends on:

* https://github.com/mozilla/fxa-auth-db-mysql/pull/65
* https://github.com/mozilla/fxa-auth-db-mem/pull/43
* https://github.com/mozilla/fxa-auth-server/pull/997

Adds the endpoint `POST /sessionToken/:id/update`.

### CODE REVIEW

File: docs/Server_API.md [hunk 1]
```diff
@@ -482,6 +483,56 @@ Content-Length: 2
...
+    * Body : {}
+* Status Code : 404 Not Found
+    * Conditions: if this session `tokenId` doesn't exist
+    * Content-Type : 'application/json'
+    * Body : `{"message":"Not Found"}`
+* Status Code : 500 Internal Server Error
+    * Conditions: if something goes wrong on the server
+    * Content-Type : 'application/json'
+    * Body : {"code":"InternalError","message":"...<message related to the error>..."}
+* ``
```
Comment (pdehaan) [inline docs/Server_API.md]: What's this?

Comment (philbooth) [inline docs/Server_API.md]: I have no idea, temporary insanity probably. Deleted, thanks. :smile:

File: docs/Server_API.md [hunk 2]
```diff
@@ -482,6 +483,55 @@ Content-Length: 2
...
+    * Content-Type : 'application/json'
+    * Body : {}
+* Status Code : 404 Not Found
+    * Conditions: if this session `tokenId` doesn't exist
+    * Content-Type : 'application/json'
+    * Body : `{"message":"Not Found"}`
+* Status Code : 500 Internal Server Error
+    * Conditions: if something goes wrong on the server
+    * Content-Type : 'application/json'
+    * Body : {"code":"InternalError","message":"...<message related to the error>..."}
```
Comment (pdehaan) [inline docs/Server_API.md]: nit: Not sure if you want to wrap the {"code":"", "message":""} in a \` (\<code\> tag), similar to line 529 above.

**Comment by philbooth** [comment]
Very happily closing this PR now that the repos are merged. :smile:

**Pull request closed without merging**